app/models/document.rb

```diff
@@ -347,6 +347,10 @@ def send_email_on_publish?
     update_type == "major"
   end
 
+  def urgent
```

Comment:
Might be worth to put something like https://github.com/alphagov/specialist-publisher/blob/6646ebe34f03d8ea9537c622edf83c32060ba27d/app/exporters/formatters/abstract_document_publication_alert_formatter.rb#L99-L102 here too, to carry over the knowledge of this behaviour.